.gitignore

```diff
@@ -2,3 +2,5 @@ build
 .lock*
 *.pyc
 .waf*
+sphinx
+*.code-workspace
```

Comment:
Don't add IDE-specific ignores here. They belong better in your personal .gitignore.

Reply:
Ok - thanks - I'll do that.